src/main/java/com/moneymatters/controllers/SaleController.java

```diff
@@ -0,0 +1,52 @@
+package com.moneymatters.controllers;
+
+import com.moneymatters.data.dtos.SaleDto;
+import com.moneymatters.data.models.Sale;
+import com.moneymatters.services.SaleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/sales")
+public class SaleController {
+
+    private final SaleService saleService;
+
+    @Autowired
```

Comment:
Once you already made a construct to inject de dependency, it is not necessary having @Autowired annotation

Reply:
Ok! I finished the correction feature. #17